client/cli/src/commands/purge_chain_cmd.rs

```diff
@@ -33,13 +34,23 @@ pub struct PurgeChainCmd {
 	#[allow(missing_docs)]
 	#[structopt(flatten)]
 	pub shared_params: SharedParams,
+
+	/// Select database backend to use.
+	#[structopt(
+		long = "database",
+		alias = "db",
+		value_name = "DB",
+		case_insensitive = true,
+		default_value = "RocksDb"
+	)]
+	pub database: Database,
```

Comment:
```suggestion
	/// Select database backend to use.
	#[structopt(
		long,
		alias = "db",
		value_name = "DB",
		case_insensitive = true,
		default_value = "RocksDb"
	)]
	pub database: Database,
```

Reply:
Ok so I checked the code and I think on of the problems come from this:

https://github.com/paritytech/substrate/blob/94906baab60b9dc7e71c36012b7660e3077b2d5a/client/cli/src/params/shared_params.rs#L75-L94

This function has nothing to do there as it doesn't use `self` at all. I think the code should be moved directly into there:

https://github.com/paritytech/substrate/blob/94906baab60b9dc7e71c36012b7660e3077b2d5a/client/cli/src/config.rs#L169-L182